resources/js/helpers/helpers.js

```diff
@@ -399,6 +403,48 @@ export function getInitialAttributeValue(attribute) {
     }
 }
 
+export function getEmptyAttributeValue(type) {
+    if(!type) return null;
+
+    switch(type) {
+        case 'boolean':
+            return false;
+        case 'geography':
+        case 'iconclass':
+        case 'string-sc':
+        case 'date':
+        case 'url':
+            return '';
```

Comment:
There are 3 positions where we return ' '. Combine the non-default cases that return ''?

Reply:
Yes, we can add this at the end right before `default`. Thought I already did that 🤔